internal/testrunner/runners/system/servicedeployer/terraform.go

```diff
@@ -64,6 +64,12 @@ func addTerraformOutputs(outCtxt ServiceContext) error {
 	// Unmarshall the data into `terraformOutputs`
 	logger.Debug("Unmarshalling terraform output json")
 	var terraformOutputs map[string]OutputMeta
+
+	if !json.Valid(content) {
+		logger.Debug("Invalid Json content in the terraform output file, skipped creating outputs")
+		return nil
+	}
```

Comment:
What happens if `aws_logs` has an invalid json here? Would the test package fail if that happens ?

Reply:
The system test fails on the package if there is an invalid json because the output is not available in the system test config